central/telemetry/phonehome/interceptor.go

```diff
@@ -60,6 +61,16 @@ func getRequestDetails(ctx context.Context, err error, info *grpc.UnaryServerInf
 		method = "unknown"
 		code = -1
 	}
+
+	// By default, all permanent HTTP headers in grpc-gateway are added grpcgateway- prefix:
+	// https://github.com/grpc-ecosystem/grpc-gateway/blob/master/runtime/mux.go#L106-L114
```

Comment:
```suggestion
	// https://github.com/grpc-ecosystem/grpc-gateway/blob/8952e38d5addd28308e29c272c696a578aa8ace8/runtime/mux.go#L106-L114
```
Always press `y` in GitHub to get a link with pinned line references.